react/training-project/src/app/components/Button/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button(props) {
```

Comment:
For future components, you can do this:
```
function Button({move, children}) {
...
// Use them like {children} or {move} instead of {props.children} or {props.move}
}
```